addons/website_sale/static/src/js/website_sale_tour.js

```diff
@@ -0,0 +1,481 @@
+                });
+            });
+            return Promise.all([taxDef, def2]).then(function (res) {
+                window.location.href = '/web/session/logout';
+            });
+        },
+    },
+    // Testing b2c with Tax-Excluded Prices
+    {
+        content: "Click on shop",
+        extra_trigger: '.oe_login_form',
+        trigger: '#top_menu a[href="/shop"]:visible',
+    },
+    {
+        content: "Search product 'Storage box'",
+        extra_trigger: '.js_sale',
+        trigger: '#wrapwrap',
+        run: function () {
+            window.location.href = '/shop?search=Storage Box';
+        },
+    },
+    {
+        content: "Open product page",
+        trigger: '.oe_product_cart a:contains("Storage Box")',
+    },
+    {
+        content: "Add one more storage box",
+        trigger: '.js_add_cart_json:eq(1)',
+    },
+    {
+        content: "Check b2b Tax-Excluded Prices",
+        trigger: '.product_price .oe_price .oe_currency_value:containsExact(79.00)',
+    },
+    {
+        content: "Click on add to cart",
+        trigger: '#add_to_cart',
+    },
+    {
+        content: "Check for 2 products in cart and proceed to checkout",
+        extra_trigger: '#cart_products tr:contains("Storage Box") input.js_quantity:propValue(2)',
+        trigger: 'a[href*="/shop/checkout"]',
+    },
+    {
+        content: "Check Price b2b subtotal",
+        trigger: 'tr#order_total_untaxed .oe_currency_value:containsExact(158.00)',
+    },
+    {
+        content: "Check Price b2b Sale Tax(15%)",
+        trigger: 'tr#order_total_taxes .oe_currency_value:containsExact(23.70)',
+    },
+    {
+        content: "Check Price b2b Total amount",
+        trigger: 'tr#order_total .oe_currency_value:containsExact(181.70)',
+    },
+    {
+        content: "Writing name in address",
+        trigger: 'input[name="name"]',
+        run: 'text abc',
+    },
+    {
+        content: "Writing phone number in address",
+        trigger: 'input[name="phone"]',
+        run: 'text 9999999999',
+    },
+    {
+        content: "Writing email in address",
+        trigger: 'input[name="email"]',
+        run: 'text abc@odoo.com',
+    },
+    {
+        content: "Writing street and Number in address",
+        trigger: 'input[name="street"]',
+        run: 'text 1,C.G Road',
+    },
+    {
+        content: "Writing city in address",
+        trigger: 'input[name="city"]',
+        run: 'text xyz',
+    },
+    {
+        content: "Select country in address",
+        trigger: 'select[name="country_id"]',
+        run : function (actions) {
+            $('#country_id option:eq(1)').attr('selected', true);
+        }
+    },
+    {
+        content: "Shipping address is not same as billing address",
+        trigger: '#shipping_use_same',
+    },
+    {
+        content: "Click on next button",
+        trigger: '.oe_cart .btn:contains("Next")',
+    },
+    {
+        content: "Writing name in shipping address",
```

Comment:
Same remark, these 5 steps can be written in a single one to gain some time without reducing what is tested.